spec/services/contests/close_registration_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ContestsServices::CloseRegistration do
+  Delayed::Worker.delay_jobs = false
+
+  let!(:contest) { create :contest }
+
+  context 'There were users who registered for the contest' do
+    let!(:user1) { create :user }
+    let!(:user2) { create :user }
+    let!(:user3) { create :user }
+    let!(:contest_application1) { create :contest_application, user_id: user1.id, contest_id: contest.id }
+    let!(:contest_application2) { create :contest_application, user_id: user2.id, contest_id: contest.id }
+    let!(:contest_application3) { create :contest_application, user_id: user3.id, contest_id: contest.id }
```

Comment:
Так как далее user1, user2 и user3 не используются, можно отдать их на автогенерацию FactoryBot'у.
```suggestion
    let!(:contest_application1) { create :contest_application, contest_id: contest.id }
    let!(:contest_application2) { create :contest_application, contest_id: contest.id }
    let!(:contest_application3) { create :contest_application, contest_id: contest.id }
```